README.md

```diff
@@ -1,23 +1,70 @@
-# Devito: Fast Finite Difference Computation from Symbolic Specification
+# Devito: Fast Tensor Computation from Symbolic Specification
```

Comment:
Do you really want to completely drop FD?

Reply:
IF we drop FD I would say stencil, not Tensor.

Reply:
> Do you really want to completely drop FD?

No, it's mentioned all over the place

> IF we drop FD I would say stencil, not Tensor.

I don't mind stencil, but I'd want the others' opinion on this

Reply:
OK, using "stencil" for now